src/test/librbd/deep_copy/test_mock_SetHeadRequest.cc

```diff
@@ -71,6 +72,28 @@ class DetachParentRequest<MockTestImageCtx> {
 
 DetachParentRequest<MockTestImageCtx> *DetachParentRequest<MockTestImageCtx>::s_instance;
 
+template <>
+class DetachChildRequest<MockTestImageCtx> {
```

Comment:
What is this template specialization for?  It doesn't seem like any changes to `src/test/librbd/deep_copy/test_mock_SetHeadRequest.cc` are needed.